### PR TITLE
feat: api alignment AmityCreateStoryPage

### PR DESCRIPTION
**Pull request by NaingAmity**

### CODE REVIEW

File: src/v4/PublicApi/AmityCreateStoryPage/AmityCreateStoryPage.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+const AmityCreateStoryPage: FC<ICreateStoryPage> = ({
+  targetId,
+  targetType,
+}) => {
+  const { getImage } = useFile();
+  const navigation =
+    useNavigation() as NativeStackNavigationProp<RootStackParamList>;
+  const [isLoading, setIsLoading] = useState(false);
+  console.log('aa');
```
Comment (bonanaaaaaa) [inline src/v4/PublicApi/AmityCreateStoryPage/AmityCreateStoryPage.tsx:19]: Remove this line

File: src/v4/PublicApi/AmityCreateStoryPage/AmityCreateStoryPage.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  useLayoutEffect(() => {
+    console.log('Gg');
```
Comment (bonanaaaaaa) [inline src/v4/PublicApi/AmityCreateStoryPage/AmityCreateStoryPage.tsx:21]: This one too

Comment (NaingAmity) [reply src/v4/PublicApi/AmityCreateStoryPage/AmityCreateStoryPage.tsx]: Removed krub

File: src/v4/PublicApi/AmityCreateStoryPage/AmityCreateStoryPage.tsx (hunk 1 above)
Comment (NaingAmity) [reply src/v4/PublicApi/AmityCreateStoryPage/AmityCreateStoryPage.tsx]: Removed krub

**Pull request closed without merging**